### ISSUE TITLE
Stop tests on push

### ISSUE THREAD

**Issue by RitikJaiswal75** [Real-Dev-Squad/website-crypto#283]
## Issue
- When a PR is created, the git workflow runs tests on all the pull requests, but once the tests are run on pull requests they run again during push

### When was the issue discovered?
- The issue was discovered on 18th August 2022 during a hangout call while trying to merge PRs.

### Files that might be creating this
- `.github/workflows/test.yml`

### Screenshot of the Problem
![image](https://user-images.githubusercontent.com/57758447/185927191-4f7245d6-44b9-46a7-a355-b49ec54ef8b8.png)

**Comment by bajajcodes** [comment]
<s>Hey @RitikJaiswal75 can you assign this issue to me?</s>
Assigned, and PR raised.

**Comment by MehulKChaudhari** [comment]
Hey, I think it's good that the tests are run after every commit. It gives us an idea of which commits made the tests fail.

**Comment by bajajcodes** [comment]
> Hey, I think it's good that the tests are run after every commit. It gives us an idea of which commits made the tests fail.

This can be done locally as well, if TDD is followed.

**Comment by MehulKChaudhari** [comment]
> > Hey, I think it's good that the tests are run after every commit. It gives us an idea of which commits made the tests fail.
> 
> This can be done locally as well, if TDD is followed.

Yes, But what if developer didn't fix the tests which are breaking or didn't run tests locally? how will reviewer know whether there are breaking changes or not?
